core/templates/pages/exploration-editor-page/services/exploration-save.service.ts

```diff
@@ -205,7 +205,7 @@ angular.module('oppia').factory('ExplorationSaveService', [
             RouterService.onRefreshVersionHistory.emit({
               forceRefresh: true
             });
-            AlertsService.addSuccessMessage('Changes saved.');
+            AlertsService.addSuccessMessage('Changes saved.', 5000);
```

Comment:
How much it was earlier?
Also, is it a general toast message issue or just specific to exp? if the latter one then why don't other places have this issue? (Just curious!)

Reply:
It was 1.5s which, I thought was sufficient, but for some reason, when the network is throttled, the visibilityOf function seems to fail to catch the toast message even if it does show up on the screen. i.e. it just times out even if the toast shows.

Reply:
I am not sure if this is a general issue because I haven't seen it yet in other places but I think it is worth trying throttling the network in other places to verify.